test/test_utils.hpp

```diff
@@ -127,6 +128,22 @@ std::string get_pathname(
     return fname;
 }
 
+//****************************************************************************
+template <class BufferT>
+BufferT read_yolo_data(std::string const &data_file) {
```

Comment:
I added this in before I switched to @mcmillan03 format of binary files. Do we want to keep it in?

Reply:
Theoretically, this function will be able to read any binary file and return the data in a BufferT.

Reply:
Maybe leave it in but make function name more generic like read_binary_data(), and add a comment about the requirements/expectations of the data layout.

Also if this really only supports floating point values we may want to remove the template.  If so, add a @todo comment to do that I can address when I merge float and quint8 platforms into one platform.